error cairocffi

@pgiraud Since this morning we got the following error during the compilation. Do you have an idea where does it come from ? 

> Requirement already satisfied (use --upgrade to upgrade): cffi>=0.6 in ./.build/venv/lib/python2.7/site-packages (from weasyprint==0.23->geoportailv3==1.0->-r CONST_requirements.txt (line 9))
Collecting cairocffi>=0.5 (from weasyprint==0.23->geoportailv3==1.0->-r CONST_requirements.txt (line 9))
  Using cached cairocffi-0.7.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-X3pZpc/cairocffi/setup.py", line 65, in <module>
        **cffi_args
      File "/usr/lib64/python2.7/distutils/core.py", line 112, in setup
        _setup_distribution = dist = klass(attrs)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/setuptools/dist.py", line 272, in __init__
        _Distribution.__init__(self,attrs)
      File "/usr/lib64/python2.7/distutils/dist.py", line 287, in __init__
        self.finalize_options()
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/setuptools/dist.py", line 327, in finalize_options
        ep.load()(self, ep.name, value)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/setuptools_ext.py", line 181, in cffi_modules
        add_cffi_module(dist, cffi_module)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/setuptools_ext.py", line 48, in add_cffi_module
        execfile(build_file_name, mod_vars)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/setuptools_ext.py", line 24, in execfile
        exec(code, glob, glob)
      File "cairocffi/ffi_build.py", line 30, in <module>
        ffi.cdef(constants._CAIRO_HEADERS)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/api.py", line 105, in cdef
        self._cdef(csource, override=override, packed=packed)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/api.py", line 119, in _cdef
        self._parser.parse(csource, override=override, **options)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/cparser.py", line 299, in parse
        self._internal_parse(csource)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/cparser.py", line 304, in _internal_parse
        ast, macros, csource = self._parse(csource)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/cparser.py", line 260, in _parse
        ast = _get_parser().parse(csource)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/cffi/cparser.py", line 40, in _get_parser
        _parser_cache = pycparser.CParser()
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/pycparser/c_parser.py", line 87, in __init__
        outputdir=taboutputdir)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/pycparser/c_lexer.py", line 66, in build
        self.lexer = lex.lex(object=self, **kwargs)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/pycparser/ply/lex.py", line 911, in lex
        lexobj.readtab(lextab, ldict)
      File "/home/mire/geoportailv3/.build/venv/lib/python2.7/site-packages/pycparser/ply/lex.py", line 233, in readtab
        titem.append((re.compile(pat, lextab._lexreflags | re.VERBOSE), _names_to_funcs(func_name, fdict)))
      File "/home/mire/geoportailv3/.build/venv/lib64/python2.7/re.py", line 190, in compile
        return _compile(pattern, flags)
      File "/home/mire/geoportailv3/.build/venv/lib64/python2.7/re.py", line 240, in _compile
        p = sre_compile.compile(pattern, flags)
      File "/home/mire/geoportailv3/.build/venv/lib64/python2.7/sre_compile.py", line 512, in compile
        "sorry, but this version only supports 100 named groups"
    AssertionError: sorry, but this version only supports 100 named groups

--- a/CONST_Makefile
+++ b/CONST_Makefile
@@ -18,7 +18,7 @@ TILECLOUD_CHAIN ?= TRUE
 PRINT_VERSION ?= 3
 
 PIP_CMD ?= $(VENV_BIN)/pip
-PIP_INSTALL_ARGS += install --trusted-host pypi.camptocamp.net
+PIP_INSTALL_ARGS += install --trusted-host pypi.camptocamp.net --no-binary pycparser
 
 GIT_MODULES_FOLDER ?= .git/modules/
 

